data/notices.json

```
@@ -1,5 +1,21 @@
 {
   "notices": [
+    {
+      "title": "triggers: Deployment of trigger functions fails with a type error",
+      "issueNumber": 23407,
+      "overview": "Users of Triggers and TriggerFunctions will receive deployment errors from CloudFormation when upgrading.",
```

The phrasing is a bit awkward. How about this? 
```suggestion
      "overview": "The Trigger and TriggerFunction constructs fail to deploy when upgrading with  a type error from CloudFormation.",
```